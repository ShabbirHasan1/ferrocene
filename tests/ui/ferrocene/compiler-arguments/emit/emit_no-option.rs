// check-fail
// compile-flags: --emit=

fn main() {}

// ferrocene-annotations: um_rustc_emit
