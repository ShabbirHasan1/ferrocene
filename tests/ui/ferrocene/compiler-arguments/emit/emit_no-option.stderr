error: unknown emission type: `` - expected one of: `llvm-bc`, `asm`, `llvm-ir`, `mir`, `obj`, `metadata`, `link`, `dep-info`

